doc/user/content/releases/v0.49.md

```diff
@@ -1,13 +1,51 @@
 ---
 title: "Materialize v0.49"
 date: 2023-04-05
-released: false
+released: true
+patch: 2
 ---
 
-{{< warning >}}
-This version of Materialize is not yet released.
-{{< /warning >}}
-
 ## v0.49.0
 
-* No documented changes yet.
+#### SQL
+
+* Change the type of the following system catalog replica ID columns from integer to string:
+
+    * [`mz_catalog.mz_cluster_replicas.id`](/sql/system-catalog/mz_catalog/#mz_cluster_replicas)
+    * [`mz_internal.mz_cluster_replica_statuses.replica_id`](/sql/system-catalog/mz_internal/#mz_cluster_replica_statuses)
+    * [`mz_internal.mz_cluster_replica_heartbeats.replica_id`](/sql/system-catalog/mz_internal/#mz_cluster_replica_heartbeats)
+    * [`mz_internal.mz_cluster_replica_metrics.replica_id`](/sql/system-catalog/mz_internal/#mz_cluster_replica_metrics)
+    * [`mz_internal.mz_cluster_replica_frontiers.replica_id`](/sql/system-catalog/mz_internal/#mz_cluster_replica_frontiers)
+
+    This is part of the work to introduce system replicas, which Materialize
+    will use for verification and testing purposes and which will not affect
+    user billing or system limits {{% gh 11579 %}}. Note that, since
+    `mz_catalog` is part of Materialize’s stable interface, the change to
+    `mz_catalog.mz_cluster_replicas.id` is a [**breaking change**](/releases/#backwards-compatibility).
+    If this change causes you friction, please [let us know](https://support.materialize.com).
+
+* Add the [`ALTER OWNER`](/sql/alter-owner/) command, which updates the owner
+  of an object. This is part of the work to enable **Role-based access
+  control** (RBAC){{% gh 11579 %}}.
+
+* Add permission checks based on object ownership. To `DROP` or `ALTER` an
+  object, the executing role must now be an owner of that object or a
+  superuser. This is part of the work to enable **Role-based access control**
+  (RBAC){{% gh 11579 %}}.
+
+* Apply `PRIMARY KEY`, `UNIQUE`, and `NOT NULL` constraints to tables ingested
+  from PostgreSQL sources.
+
+* Rename [replica introspection views](https://materialize.com/docs/sql/system-catalog/mz_internal/#replica-introspection-relations)
+  for consistency, and use the `_per_worker` name suffix for per-worker introspection views.
+
+* Automatically restart failed SSH tunnels to improve the reliability of
+  SSH-tunneled Kafka sources.
+
+#### Bug fixes and other improvements
+
+- Fix a correctness bug in top-k processing for monotonic, append-only sources.
```

Comment:
Nit: I think we styleize as "Top K" elsewhere in the docs.

```suggestion
- Fix a correctness bug in Top K processing for monotonic, append-only sources.
```

Reply:
Done. TFTR.